frac_vel : 1
